packages/jalv-select/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=jalv-select
+pkgver=1.3
+pkgrel=3
+pkgdesc='A small GUI to select installed LV2 plugins and run them with jalv.'
+arch=(x86_64 aarch64)
+url='https://github.com/brummer10/jalv_select'
+license=('custom:Public Domain')
+depends=(gtkmm jalv)
+makedepends=(lilv serd sord sratom)
+source=("$pkgname-$pkgver.tar.gz::https://github.com/brummer10/${pkgname/-/_}/archive/v$pkgver.tar.gz")
+sha256sums=('13227087f107e35ecf51897ed63050c5ba6df1b442ffb2f7e876c46fb0fe11d7')
+
+prepare() {
+  cd ${pkgname/-/_}-${pkgver}
+  # Don't compress man pages, that's makepkg's job
```

Comment:
I see other packages also do this and thought this was fine. Is it considered bad practice in a Makefile?

Reply:
It led to non-reproducible build.

Reply:
See here: https://ci.cbix.de/osam-cologne/archlinux-proaudio/329/1/9